doc/apiary/v2/fiware-ngsiv2-reference.apib

```diff
@@ -364,30 +371,37 @@ Response code:
             Location: /v2/entities/Bcn-Welt
 
 
-### Create entity. Error [POST /v2/entities]
+## ContextEntity by ID [/v2/entities/{entityId}{?attrs,options}]
 
-+ Request (application/json)
-
-        {
-            "type": "Room",
-            "id": "Bcn-Welt",
-            "temperature": 21.7
-        }
+### Retrieve entity [GET /v2/entities/{entityId}{?attrs}]
 
-+ Response 422 (application/json)
+<table style="width: 100%; margin: 12px 0 0 0;">
+<tr><td colspan="2"><strong>Required Parameters</strong></td></tr>
+<tr><td colspan="2">none</td></tr>
+<tr><td colspan="2"><strong>Optional Parameters</strong></td></tr>
+<tr>
+  <td style="padding-right: 40px; width: 140px;">attrs</td>
+  <td>Comma-separated list of attribute names which will be included in the response. If this parameter is not
+  included, all the attributes are retrieved.</td>
+</tr>
+</table>
 
-        {
-            "error": "AlreadyExists",
-            "description": "An entity with the same id and type is already present"
-        }
+The response is an object representing the entity identified by the ID. The object follows
+the JSON entity representation format (described in a section above).
 
+This operation must return only one entity element, but it may happen that there are more
+than one entity with the same ID (e.g. entities with same ID but different type). In those cases
+an error mesage is returned, specifying in the description the URL that could be used to get
```

Comment:
specifying in the error payload 

I'm wondering if the payload could include in that case a xx_url field but that would be part of separate discussion